README.md

```diff
@@ -1,2 +1,45 @@
-# new-relic
-Best practice New Relic configurations.
+# New Relic considerations
+
+Considerations for using New Relic in the federal government.
+
+Below is a description of the settings in your New Relic `.ini` file that concern privacy and security. Example configuration files can be found in this repository:
+
+* [`python-low-security.ini`](python-low-security.ini)
```

Comment:
@LindsayYoung can speak to why this was needed, or how we could best specify only the things that need to be changed, vs the entire file.